scylla-gdb.py

```diff
@@ -4410,13 +4410,17 @@ def invoke(self, arg, from_tty):
         for table in all_tables(db):
             gdb.write('table %s:\n' % schema_ptr(table['_schema']).table_name())
             try:
-                for cg_ptr in std_vector(table["_compaction_groups"]):
+                for cg_ptr in chunked_vector(table["_compaction_groups"]):
                     scylla_memtables.dump_compaction_group_memtables(std_unique_ptr(cg_ptr).get())
             except gdb.error:
                 try:
-                    scylla_memtables.dump_compaction_group_memtables(std_unique_ptr(table["_compaction_group"]).get())
+                    for cg_ptr in std_vector(table["_compaction_groups"]):
+                        scylla_memtables.dump_compaction_group_memtables(std_unique_ptr(cg_ptr).get())
                 except gdb.error:
-                    scylla_memtables.dump_memtable_list(seastar_lw_shared_ptr(table['_memtables']).get()) # Scylla 5.1 compatibility
+                    try:
+                        scylla_memtables.dump_compaction_group_memtables(std_unique_ptr(table["_compaction_group"]).get())
+                    except gdb.error:
+                        scylla_memtables.dump_memtable_list(seastar_lw_shared_ptr(table['_memtables']).get()) # Scylla 5.1 compatibility
```

Comment:
as a followup, maybe we can add a generic helper to scylla-gdb that would get a list of alternative functions to extract date items like the memtable ptr in a backward compatible way and it will try them in order while gdb.error is thrown.

Cc @denesb